src/Symfony/Component/Messenger/Middleware/HandleMessageMiddleware.php

```diff
@@ -35,6 +39,7 @@ class HandleMessageMiddleware implements MiddlewareInterface
     public function __construct(
         private HandlersLocatorInterface $handlersLocator,
         private bool $allowNoHandlers = false,
+        private ?EventDispatcherInterface $eventDispatcher = null,
```

Comment:
```suggestion
        private ?EventDispatcherInterface $dispatcher = null,
```

I like it more, but lets see what the others say

Reply:
Actually `SendMessageMiddleware` [uses `$eventDispatcher`](https://github.com/symfony/symfony/blob/7.0/src/Symfony/Component/Messenger/Middleware/SendMessageMiddleware.php#L33), so I guess it's better to keep it consistent here?

Reply:
[eventDispatcher](https://github.com/search?q=repo%3Asymfony%2Fsymfony+%22EventDispatcherInterface+%24eventDispatcher%22+language%3APHP&type=code) vs [dispatcher](https://github.com/search?q=repo%3Asymfony%2Fsymfony+%22EventDispatcherInterface+%24dispatcher%22+language%3APHP&type=code&l=PHP)

but we dont mind :D